### PR TITLE
feat: dual stack (IPV4_IPV6) support

### PR DESCRIPTION
**Pull request by ghabian**
making a separate PR to attempt to fix issues with #1672 adding support for dual stack GKE clusters (by @jmymy ) 
fixes:
#1656
#1602

### CODE REVIEW

**Comment by ghabian** [comment]
note:  The PR is in draft I'm just making sure the new URL for the ACM policy bundle is correct (cc: @apeabody ). otherwise the code should be ready for review (if one is needed)

File: examples/simple_zonal_with_acm/acm.tf [hunk 1]
```diff
@@ -28,7 +28,7 @@ module "acm" {
 
   secret_type = "ssh"
 
-  policy_bundles = ["https://github.com/GoogleCloudPlatform/acm-policy-controller-library/bundles/policy-essentials-v2022?ref=981cdf31878b886b53decdade23d8f76e80140fc"]
+  policy_bundles = ["https://github.com/GoogleCloudPlatform/gke-policy-library/tree/main/bundles/policy-essentials-v2022"]
```
Comment (apeabody) [inline examples/simple_zonal_with_acm/acm.tf]: HI @ghabian - Not tested, but I suspect you will want `"https://github.com/GoogleCloudPlatform/gke-policy-library/bundles/policy-essentials-v2022?ref=7a5f87d7e1a2f63582505d14c2c90f496bcf263d"`

Comment (apeabody) [reply examples/simple_zonal_with_acm/acm.tf]: Tested as good:

```
kubectl apply -k https://github.com/GoogleCloudPlatform/gke-policy-library/bundles/policy-essentials-v2022?ref=7a5f87d7e1a2f63582505d14c2c90f496bcf263d
k8snoenvvarsecrets.constraints.gatekeeper.sh/policy-essentials-v2022-no-secrets-as-env-vars created
k8spspallowprivilegeescalationcontainer.constraints.gatekeeper.sh/policy-essentials-v2022-psp-allow-privilege-escalation created
k8spspallowedusers.constraints.gatekeeper.sh/policy-essentials-v2022-psp-pods-must-run-as-nonroot created
k8spspcapabilities.constraints.gatekeeper.sh/policy-essentials-v2022-psp-capabilities created
k8spsphostnamespace.constraints.gatekeeper.sh/policy-essentials-v2022-psp-host-namespace created
k8spsphostnetworkingports.constraints.gatekeeper.sh/policy-essentials-v2022-psp-host-network-ports created
k8spspprivilegedcontainer.constraints.gatekeeper.sh/policy-essentials-v2022-psp-privileged-container created
k8spspseccomp.constraints.gatekeeper.sh/policy-essentials-v2022-psp-seccomp-default created
k8spodsrequiresecuritycontext.constraints.gatekeeper.sh/policy-essentials-v2022-pods-require-security-context created
k8sprohibitrolewildcardaccess.constraints.gatekeeper.sh/policy-essentials-v2022-prohibit-role-wildcard-access created
k8srestrictrolebindings.constraints.gatekeeper.sh/policy-essentials-v2022-restrict-clusteradmin-rolebindings created
```

Comment (ghabian) [reply examples/simple_zonal_with_acm/acm.tf]: thanks! running the simple_zonal test locally works as well with this URL.

**Review by apeabody — APPROVED** [review]
Thanks for the contribution @ghabian!

**Pull request merged**